packages/visual-editor/src/components/ReadOnlyRenderer.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```

Comment:
[5] This renderer will render the blocks directly, as we don't need the editor things like the dropzone and hover stuff.